src/metabase/server/middleware/embedding_sdk_bundle.clj

```diff
@@ -36,6 +36,7 @@
   (fn [request]
     (let [base (response/resource-response bundle-resource)]
       (cond
-        (nil? base)        (not-found)
-        config/is-prod?    (serve-for-prod base request)
-        :else              (serve-for-dev base)))))
+        (nil? base)                         (not-found)
+        (config/config-bool :mb-is-cypress) (serve-for-dev base)
```

Comment:
Also serve no-cache version on Cypress to allow intercept to work since cached bundle [doesn't go through Cypress `intercept`.](https://docs.cypress.io/api/commands/intercept#cyintercept-and-request-caching)